gateway/gateway.go

```diff
@@ -146,6 +148,13 @@ func (g *gateway) Reconfigure(ctx context.Context, deps resource.Dependencies, c
 		g.started = false
 	}
 
+	// capture C log output
+	loggingStarted, ok := LoggingRoutineStarted[g.Name().Name]
+	if !ok || !loggingStarted {
+		g.workers = utils.NewBackgroundStoppableWorkers(g.captureCOutputToLogs)
+		LoggingRoutineStarted[g.Name().Name] = true
+	}
+
 	// maintain devices and lastReadings through reconfigure.
 	if g.devices == nil {
 		g.devices = make(map[string]*node.Node)
```

Comment:
So at this point, we just sat without a clogger?

Reply:
not sure I understand the question, can you elaborate